Q-1 :  How many except statements can a try-except block have? Name Some built-in exception classes:

=> An Exception is an Unexpected Event, which occurs during the execution of the program.   
=> It is also known as a run time error.
=> Exception handling with try, except, else, and finally block : 
    1.  Try :  This block will test the excepted error to occur
    2.  Except  :  Here you can handle the error
    3.  Else   :   If there is no exception then this block will be executed
    4.  Finally   :    Finally block always gets executed either exception is generated or not

=> Example :

def divide(x, y): 
	try: 
		result = x // y 
	except ZeroDivisionError: 
		print("Sorry ! You are dividing by zero ") 
	else:
		print("Yeah ! Your answer is :", result) 
	finally: 
		# this block is always executed 
		# regardless of exception generation. 
		print('This is always executed') 
 
divide(3, 2) 
divide(3, 0)
